app/src/components/widgets/charts/launchExecutionAndIssueStatistics/chartUtils.jsx

```diff
@@ -0,0 +1,61 @@
+import { defineMessages } from 'react-intl';
+
+const unique = (array, propName) =>
+  array.filter((e, i) => array.findIndex((a) => a[propName] === e[propName]) === i);
+
+export const getPercentage = (value) => (value * 100).toFixed(2);
+export const getDefectItems = (items) =>
+  unique(
+    items.map((item) => ({
+      id: item[0],
+      count: item[1],
+      name: item[0]
+        .split('$')
+        .slice(0, 3)
+        .join('$'),
+    })),
+    'name',
+  );
+
+export const messages = defineMessages({
```

Comment:
Please use some of our utilities and messages defined in `components/widgets/charts/common`